.pipelines/templates/compliance.yml

```diff
@@ -0,0 +1,46 @@
+# This Yaml Document has been converted by ESAI Yaml Pipeline Conversion Tool.
+# Please make sure to check all the converted content, it is your team's responsibility to make sure that the pipeline is still valid and functions as expected.
+parameters:
+  parentJobs: []
+jobs:
+- job: compliance
+  variables:
+  - name: runCodesignValidationInjection
+    value: false
+  - name: NugetSecurityAnalysisWarningLevel
+    value: none
+  - name: ob_outputDirectory
+    value: '$(Build.ArtifactStagingDirectory)/ONEBRANCH_ARTIFACT'
+  displayName: Compliance
+  dependsOn: ${{ parameters.parentJobs }}
+  pool:
+    type: windows
+  steps:
+  - checkout: self
+    clean: true
+  - template: /.pipelines/templates/SetVersionVariables.yml@self
+    parameters:
+      ReleaseTagVar: $(ReleaseTagVar)
+  - task: DownloadBuildArtifacts@0
+    displayName: 'Download artifacts'
+    inputs:
+      buildType: current
+      downloadType: single
+      artifactName: results
+      downloadPath: '$(System.ArtifactsDirectory)'
+  - powershell: |
+      dir "$(System.ArtifactsDirectory)\*" -Recurse
+    displayName: 'Capture artifacts directory'
+    continueOnError: true
+  - template: /.pipelines/templates/expand-compliance.yml@self
+    parameters:
+      architecture: fxdependent
+      version: $(version)
+  - template: /.pipelines/templates/expand-compliance.yml@self
+    parameters:
+      architecture: x86
+      version: $(version)
+  - template: /.pipelines/templates/expand-compliance.yml@self
```

Comment:
does this do anything?  Is it supposed to?

Reply:
These files are going to be replaced with packaging changes